backend-shared/config/db/db_field_constants.go

```diff
@@ -90,3 +90,90 @@ func TruncateVarchar(s string, maxLength int) string {
+	"KubernetesToDBResourceMappingDBRelationType":                       KubernetesToDBResourceMappingDbRelationTypeLength,
+	"KubernetesToDBResourceMappingDbRelationKey":                        KubernetesToDBResourceMappingDbRelationKeyLength,
+	"KubernetesToDBResourceMappingDBRelationKey":                        KubernetesToDBResourceMappingDbRelationKeyLength,
+	"APICRToDatabaseMappingApiResourceType":                             APICRToDatabaseMappingApiResourceTypeLength,
+	"APICRToDatabaseMappingAPIResourceType":                             APICRToDatabaseMappingApiResourceTypeLength,
+	"APICRToDatabaseMappingApiResourceUID":                              APICRToDatabaseMappingApiResourceUIDLength,
+	"APICRToDatabaseMappingAPIResourceUID":                              APICRToDatabaseMappingApiResourceUIDLength,
+	"APICRToDatabaseMappingApiResourceName":                             APICRToDatabaseMappingApiResourceNameLength,
+	"APICRToDatabaseMappingAPIResourceName":                             APICRToDatabaseMappingApiResourceNameLength,
+	"APICRToDatabaseMappingApiResourceNamespace":                        APICRToDatabaseMappingApiResourceNamespaceLength,
+	"APICRToDatabaseMappingAPIResourceNamespace":                        APICRToDatabaseMappingApiResourceNamespaceLength,
+	"APICRToDatabaseMappingApiResourceWorkspaceUID":                     APICRToDatabaseMappingApiResourceWorkspaceUIDLength,
+	"APICRToDatabaseMappingWorkspaceUID":                                APICRToDatabaseMappingApiResourceWorkspaceUIDLength,
+	"APICRToDatabaseMappingDbRelationType":                              APICRToDatabaseMappingDbRelationTypeLength,
+	"APICRToDatabaseMappingDBRelationType":                              APICRToDatabaseMappingDbRelationTypeLength,
+	"APICRToDatabaseMappingDbRelationKey":                               APICRToDatabaseMappingDbRelationKeyLength,
+	"APICRToDatabaseMappingDBRelationKey":                               APICRToDatabaseMappingDbRelationKeyLength,
+	"SyncOperationSyncoperationID":                                      SyncOperationSyncoperationIDLength,
+	"SyncOperationSyncOperationID":                                      SyncOperationSyncoperationIDLength,
+	"SyncOperationApplicationID":                                        SyncOperationApplicationIDLength,
+	"SyncOperationOperationID":                                          SyncOperationOperationIDLength,
+	"SyncOperationDeploymentName":                                       SyncOperationDeploymentNameLength,
+	"SyncOperationDeploymentNameField":                                  SyncOperationDeploymentNameLength,
+	"SyncOperationRevision":                                             SyncOperationRevisionLength,
+	"SyncOperationDesiredState":                                         SyncOperationDesiredStateLength,
+}
+
+// Get value of constants based on constant variable name given as String.
+func GetConstantValue(variable string) int {
```

Comment:
Since this function isn't used by any other packages (AFAICT), it doesn't need to be exported, and can be renamed to `getConstantValue`.